src/utils/common.js

```diff
@@ -7,7 +7,10 @@ export default {
     },
     checkPrecision(amount, unity) {
         // return false if check failed
-        return !(BigNumber(amount).toString().split('.')[1] && BigNumber(amount).toString().split('.')[1].length > unity)
+        amount = BigNumber(amount)
+        let m = amount.toExponential().match(/\d(?:\.(\d*))?e([+-]\d+)/)
+        amount = amount.toFixed(Math.max(0, (m[1] || '').length - m[2]))
+        return !(amount.toString().split('.')[1] && amount.toString().split('.')[1].length > unity)
```

Comment:
When using `checkPrecision`, unity is `undefined` value here. No need change logic of this part. Fix `checkPrecision` issue in other PR.